src/index.ts

```diff
@@ -24,6 +24,39 @@ type LooseNode = Node & {
   value: any;
 };
 
+const generateHeaders = async () => {
+  await writeFile(
+    ".vercel/output/static/_headers",
+    `\n/_next/static/*\n  Cache-Control: public,max-age=31536000,immutable\n`,
+    {
+      // in case someone configured redirects already, append to the end
+      flag: "a",
+    }
+  );
+};
+
+const generateRoutes = async () => {
+  try {
+    await writeFile(
+      ".vercel/output/static/_routes.json",
+      JSON.stringify({
+        version: 1,
+        include: ["/*"],
+        exclude: ["/_next/static/*"],
+      }),
+      { flag: "ax" } // don't generate file if it's already manually maintained
+    );
+  } catch (e) {
+    if (e.code != "EEXIST") {
+      throw e;
+    }
+  }
+};
+
+const generateMetadataFromConfig = async () => {
```

Comment:
I'm not sure why its `FromConfig` since it just generated the data without any configuration?

Reply:
Ah, that function used to actually use the config, but I dropped it since it wasn't really necessary.